CollapseLauncher/Classes/Properties/ArgumentParser.cs

```diff
@@ -207,6 +211,33 @@ public static void ParseMoveSteamArguments(params string[] args)
             var rootCommand = new RootCommand();
             rootCommand.AddCommand(command);
         }
+
+        private static void ParseStartGameArguments(params string[] args)
+        {
+            var gameOption = new Option<string>(new string[] { "--game", "-g" }, description: "Game") { IsRequired = true };
+            var regionOption = new Option<string>(new string[] { "--region", "-r" }, description: "Region") { IsRequired = false };
+            var startGameOption = new Option<bool>(new string[] { "--play", "-p" }, description: "Start Game after loading the Region") { IsRequired = false };
+            var command = new Command("open", "Open the Launcher in a specific region");
```

Comment:
Since the argument required is the --game, might want to change this to "into specific game" or "into specific game/region"